src-php/Providers/PackageServiceProvider.php

```diff
@@ -61,15 +61,15 @@ private function loadMigrations()
 
     private function registerGates()
     {
-        Gate::define('accessContent', function ($user, $access = null) {
-            if ($access === null) {
+        Gate::define('accessContent', function ($user, $model = null) {
+            if ($model->access === null) {
```

Comment:
Just need to add a check here as this will trip if the model doesn't have an `access` relationship, or wrap it in an optional?